src/lib/providers/kubernetes/start.ts

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env ts-node
+import {createRoadieAgentEntityProvider, RoadieAgent} from "../../";
+import {KubernetesEntityProvider} from "@/providers/kubernetes/EntityProvider";
+import {getRootLogger} from "@/logger";
+import yaml from 'yaml';
+
+const podTemplate = `
+---
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: {{ metadata.name }}
+  namespace: {{ metadata.namespace }}
+spec:
+  type: service
+  lifecycle: unknown
+  owner: test
+`;
+
+const helmChartTemplate = `
+---
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: {{ metadata.name }}
+  namespace: {{ metadata.namespace }}
+spec:
+  type: 'helm-chart'
+  lifecycle: unknown
+  owner: test
+`;
+
+const main = async () => {
+    const logger = getRootLogger();
+    const brokerServer = process.env.BROKER_SERVER || 'http://localhost:7341';
+
+    RoadieAgent.fromConfig({
+        server: brokerServer,
```

Comment:
The settings here will need to be provided from the outside.. and some of the settings should be defaulted in the implementation of fromConfig.